src/benchmarks/real-world/Roslyn/StageBenchmarks.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Emit;
+using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.Compilation;
+
+namespace CompilerBenchmarks
+{
+    public class StageBenchmarks
+    {
+        private Compilation _comp;
+        private CommonPEModuleBuilder _moduleBeingBuilt;
+        private EmitOptions _options;
+        private MemoryStream _peStream;
```

Comment:
`_peStream` seems to be setup only in `LoadCompilation` for the `CompileMethodsAndEmit` method/benchmark. On the other hand, `_peStream` is used in the `SerializeMetadata` method/benchmark, but it's not initialized/reset in the global setup for it (the `CompileMethods` method). Is this intentional?

Reply:
The Setup method for the other one calls LoadCompilation. Should be fine.